README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
+5. When completed, the rows will disappear.
+6. To pause the game, just press the space bar again.
+
```

Comment:
```suggestion
### Instructions for playing the game

1. Press the space bar to begin.
2. Use the up and down arrow keys to rotate the shape.
3. Use the left and right arrow keys to position the shape.
4. The goal is to create complete rows with no empty spaces.
5. When completed, the rows will disappear.
6. To pause the game, just press the space bar again.

```
I think.... the **obscurity** of the passage makes several interpretations possible.🙄
But I still accept your request【